intents/intent.ridl

```diff
@@ -151,15 +151,21 @@ struct IntentDataGetIdToken
 struct TransactionRaw
   - type: string
   - to: string
-  - value: string
+  - value?: string
```

Comment:
value is now optional, will default to 0

Reply:
Maybe add `+ go.field.type = string` so that the type doesn't get turned into a pointer?

Reply:
thats a good idea, I'll do that.